ignite/cmd/scaffold_chain.go

```diff
@@ -100,7 +101,11 @@ func scaffoldChainHandler(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	if noDefaultModule && len(params) > 0 {
+	moduleConfigs, err := cmd.Flags().GetStringSlice(flagModuleConfigs)
+	if err != nil {
+		return err
+	}
+	if noDefaultModule && (len(params) > 0 || len(moduleConfigs) > 0) {
 		return errors.New("params flag is only supported if the default module is enabled")
 	}
```

Comment:
It might be helpful to add more context by handling both error cases:

```suggestion
	if noDefaultModule {
		if len(params) > 0 {
			return errors.New("params flag is only supported if the default module is enabled")
		} else if len(moduleConfigs) > 0) {
			return errors.New("module configs flag is only supported if the default module is enabled")
		}
	}
```